File: cMySQL.cs
// Program to connect with MySQL database.

using MySql.Data.MySqlClient;
using System;
using iCRUD;
using cCRUD;

class cMySQL : cCRUD.cCRUD, iCRUD.iCRUD
{
	public static void Main(string[] args) {
		MySqlConnection conn;
		string myConnectionString;
		myConnectionString = "server=138.68.140.83;uid=Sahith;" +"pwd=Sahith@369;database=dbSahith";
		try {
			conn = new MySqlConnection();
			conn.ConnectionString = myConnectionString;
			conn.Open();
			Console.WriteLine("Connected to MySQL!");
			conn.Close();
		}
		catch (MySqlException ex) {
			Console.WriteLine(ex.ToString());
		}
		Console.WriteLine("Done.");
	}
}

File: cMySQL.java
// Program to define methods for MySQL database.

import java.sql.*;
import java.util.*;

class cMySQL extends cCRUD implements iCRUD {
	cMySQL() {
		try {
			Class.forName("com.mysql.cj.jdbc.Driver");
			connection = DriverManager.getConnection("jdbc:mysql://138.68.140.83:3306/dbSahith", "Sahith", "Sahith@369");
			statement = connection.createStatement();
			System.out.println("Connected successfully!");
		}
		catch (Exception e) {
			System.out.println(e);
		}
	}
}